lib/at_monitor/Kconfig

```diff
@@ -14,8 +14,24 @@ config AT_MONITOR_HEAP_SIZE
 	range 64 4096
 	default 256
 
-config SYSTEM_WORKQUEUE_STACK_SIZE
-	default 1152 if (LTE_LINK_CONTROL && LOG)
+config AT_MONITOR_USE_DEDICATED_WORKQUEUE
+	bool "Use dedicated workqueue to dispatch notifications"
+	default y
+
+if AT_MONITOR_USE_DEDICATED_WORKQUEUE
+
+config AT_MONITOR_WORKQ_STACK_SIZE
+	int "Workqueue stack size"
+	default 768
```

Comment:
Modem shell requires 1016 bytes in the boot. I'd say that minimum default that we should put here is 1024 but should it be a bit more. I haven't been able to get mosh usage above that 1016 bytes yet.

Reply:
Should I update this value in `mosh` 's `prj.conf` then? I'd like to keep this low by default.